### DISCUSSION TITLE
Hello, everybody. Welcome.

### DISCUSSION THREAD

**Discussion by xpf0000** [xpf0000/FlyEnv#54] [General]
Here, you can talk freely, share your experience, suggest products, and ask any questions you may have.

**Comment by russellorv** [comment]
@xpf0000 thanks for your work, it would be cool to add another email testing service like MailHog

**Reply by xpf0000** [reply]
> @xpf0000 thanks for your work, it would be cool to add another email testing service like MailHog

has added in lasted version. used Mailpit.

**Comment by nethricia** [comment]
hey thank you so much for flyenv, really wonderful and great project
just want to ask quick question (sorry i'm not yet read through all the docs nor website), can i add other programming language that not mentioned on the web or docs ? like ruby, erlang or elixir ?

**Reply by xpf0000** [reply]
> hey thank you so much for flyenv, really wonderful and great project just want to ask quick question (sorry i'm not yet read through all the docs nor website), can i add other programming language that not mentioned on the web or docs ? like ruby, erlang or elixir ?

At present, users are unable to add it themselves. Erlang already has it, but it's in the mailpit module. You can create other issues Will consider adding it later
